rotkehlchen/premium/sync.py

```diff
@@ -46,7 +47,7 @@ def __init__(self, migration_manager: DataMigrationManager, data: DataHandler) -
             self.last_data_upload_ts = data.db.get_cache(
                 cursor=cursor, name=DBCache.LAST_DATA_UPLOAD_TS,
             )
-            self.last_upload_attempt_ts = self.last_data_upload_ts
+            self.last_upload_attempt_ts = Timestamp(0) if self.last_data_upload_ts is None else self.last_data_upload_ts  # noqa: E501
```

Comment:
Hmmm doesn't this also leave `self.last_data_upload_ts` as `None` in the case of `None`?

```suggestion
            self.last_data_upload_ts = Timestamp(0) if self.last_data_upload_ts is None else self.last_data_upload_ts  # noqa: E501
            self.last_upload_attempt_ts = self.last_data_upload_ts
```